addons/account_peppol/models/res_partner.py

```diff
@@ -72,7 +72,7 @@ def fields_get(self, allfields=None, attributes=None):
     def _get_participant_info(self, edi_identification):
         hash_participant = md5(edi_identification.lower().encode()).hexdigest()
         endpoint_participant = parse.quote_plus(f"iso6523-actorid-upis::{edi_identification}")
-        peppol_user = self.env.company.account_edi_proxy_client_ids.filtered(lambda user: user.proxy_type == 'peppol')
+        peppol_user = self.env.company.account_edi_proxy_client_ids.sudo().filtered(lambda user: user.proxy_type == 'peppol')
```

Comment:
tested locally, this still fails.
needs to be
```suggestion
        peppol_user = self.env.company.sudo().account_edi_proxy_client_ids.filtered(lambda user: user.proxy_type == 'peppol')
```

Reply:
interesting, didn't fail for me locally :(

Reply:
thank you for your review, you're right. I didn't realize that my demo data was reset when I restarted the db and my read-only mode was changed to billing.

as discussed, we should probably find a different approach in master (the peppol validity field should be company dependent anyway) but in stable this should be fine and it should be safe to use sudo in this case.